envergo/templates/moulinette/result.html

```diff
@@ -35,13 +35,47 @@ <h1 class="fr-mb-5w">Simulation réglementaire du projet</h1>
             aria-controls="share-modal"
             data-btn="bottom">Partager cette page par email</button>
   </div>
-
+  <div class="print-only">
+    <h2 class="fr-h3">Caractéristiques du projet</h2>
+
+    {% include '_map_snippet.html' with map=moulinette.map map_id="summary" %}
```

Comment:
this template is loaded only on print, and the map do not have enough time to be fully displayed.